src/components/widgets/accounts-invoices-list/accounts-invoices-list.directive.coffee

```diff
@@ -4,11 +4,10 @@ module.controller('WidgetAccountsInvoicesListCtrl', ($scope, $q, ImpacRoutes, Bo
   # == Context and Helpers ========================================================================
   w = $scope.widget
   bolts = ImpacRoutes.bolts()
-  bolt_path = _.find(bolts, {name: 'finance', provider: 'maestrano'}).path
+  bolt_path = w.metadata.bolt_path
```

Comment:
Are you sure this would still work when you add the widget to the dashboard the first time? Not sure the `metadata` would be set already in this case?...
What was wrong with the previous solution?